drivers/dma/dma_stm32.c

```diff
@@ -62,7 +143,13 @@ static void dma_stm32_irq_handler(void *arg)
 	}
 
 	stream = &data->streams[id];
-	stream->busy = false;
+	/*
+	 * when using the dmamux, in periph2memm or mem2periph
```

Comment:
If valid only for DMAMUX, should it be under "#ifdef CONFIG_DMAMUX_STM32"

Reply:
yes